src/Cli/Command/GenerateQRFactory.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace ConferenceTools\Checkin\Cli\Command;
+
+use Carnage\Cqorms\Persistence\ReadModel\DoctrineRepository;
+use ConferenceTools\Checkin\Domain\ReadModel\Delegate;
+use Doctrine\ORM\EntityManager;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class GenerateQRFactory implements FactoryInterface
+{
+    public function createService(ServiceLocatorInterface $serviceLocator)
+    {
+        $serviceLocator = $serviceLocator->getServiceLocator();
+        return GenerateQR::build(
```

Comment:
This is equivalent to a double new, so the doctrine repository creation should be another factory IMO.

Reply:
I plan to add an abstract factory for the repositories in the base library, so this will be resolved eventually.